src/common.py

```diff
@@ -11,6 +11,9 @@
 from src.models.wca.export import get_latest_export
 
 class Common(object):
+  
+  currentDate = datetime.datetime.now()
```

Comment:
nit: the rest of Common uses underscores (current_date), use that for consistency.